pennylane/_device.py

```diff
@@ -322,6 +322,24 @@ def execute(self, queue, observables, parameters={}, **kwargs):
 
             return self._asarray(results)
 
+    def batch_execute(self, circuits):
+        """Execute a batch of quantum circuits on the device.
+
+        The circuits are represented by tapes, and they are executed one-by-one using the
+        tape's `execute` method. This ensures that the correct interface is applied to the tape.
```

Comment:
```suggestion
        tape's ``execute`` method. This ensures that the correct interface is applied to the tape.
```

Reply:
Does this mean you can mix and match interfaces? Not that that would ever makes sense, just interesting!

Reply:
It seems so!